vmm/src/config.rs

```diff
@@ -281,11 +283,14 @@ impl<'a> FsConfig<'a> {
                 num_queues_str = &param[11..];
             } else if param.starts_with("queue_size=") {
                 queue_size_str = &param[11..];
+            } else if param.starts_with("cache_size=") {
+                cache_size_str = &param[11..];
             }
         }
 
         let mut num_queues: usize = 1;
         let mut queue_size: u16 = 1024;
+        let mut cache_size: Option<u64> = None;
```

Comment:
No default value?

Reply:
So using this command line, that has no mention about DAX, radically changes the behaviour? I'm not sure about that...